lib/tasks/participa.rake

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
```

Comment:
Can you pleas add a description of what are this rake tasks about: explain the problem in the header of the file; and the solutions that each tasks offer in each task's description.

Reply:
Done!